src/Drawer/Drawer.js

```diff
@@ -41,7 +41,8 @@ export const styleSheet = createStyleSheet('MuiDrawer', (theme) => {
       WebkitOverflowScrolling: 'touch', // Add iOS momentum scrolling.
     },
     docked: {
-      flex: '0 0 auto',
```

Comment:
Makes me wonder if we should provide some higher level layout components for application shells.